proto/osmosis/downtime-detector/v1beta1/downtime_duration.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+package osmosis.downtimedetector.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/any.proto";
+import "cosmos_proto/cosmos.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/osmosis-labs/osmosis/v13/x/downtime-detector/types";
+
+enum Downtime {
```

Comment:
I noticed you're not using `iota`. Is that so that we don't accidentally change the internal representation by reordering these?

Reply:
Yeah exactly! (Plus this is a proto file, which I don't think supports iota?)

Reply:
lol, right! didn't notice this was in the proto